README.md

```diff
@@ -203,30 +203,32 @@ event store. This increases the cost of storage as well as warehouse costs durin
 
 To mitigate this, an in-memory assignment cache is optionally available with expiration based on the least recently accessed time.
 
-It can be configured with a maximum size to fit your desired memory allocation.
+The caching can be configured individually for assignment logs and bandit action logs using `LruAssignmentLogger` and `LruBanditLogger` respectively.
+Both loggers are configured with a maximum size to fit your desired memory allocation.
 
 ```go
 import (
-  "github.com/Eppo-exp/golang-sdk/v5/eppoclient"
+	"github.com/Eppo-exp/golang-sdk/v5/eppoclient"
 )
 
 var eppoClient *eppoclient.EppoClient
 
 func main() {
-  assignmentLogger := NewExampleAssignmentLogger()
+	assignmentLogger := NewExampleAssignmentLogger()
 
-  eppoClient, _ = eppoclient.InitClient(eppoclient.Config{
-    ApiKey:           "<your_sdk_key>",
-    // 10000 is the maximum number of assignments to cache
-    // Depending on the length of your flag and subject keys, taking a median
-    // length of 32 characters, each assignment cache entry uses approximately 112 bytes.
-    // Use this calculation to determine the maximum number of assignments to cache
-    // for the memory you wish to allocate.
```

Comment:
Note: removed this memory calculation because it seems off.

Go strings have 16-byte overhead (8 for pointer + 8 for length). So 4 32-byte strings are occupying 4*(32+16) = 192 bytes. TwoQueueCache may store multiple copies of keys and values struct (string headers without actual data which can be shared, = 4*16 = 64 bytes) depending on whether entry is in frequent, recent, or recently evicted map.

So one entry (assuming 32-byte flag/subject/allocation/variation keys) occupies somewhere between 192 and 256 bytes (or likely more).